### DISCUSSION TITLE
How to filter out the contents of an array whose length is greater than 1

### DISCUSSION THREAD

**Discussion by eightHundreds** [discoveryjs/jora#41] [Q&A] [answered]
input:
```
[
  {
    arr:[1,2,3],
  },
  {
    arr:[1]
  },
  {
    arr:[2,3,4]
  }
]
```

query:
```
???
```

Expected result
```
[
    {
    arr:[1,2,3],
  },
  {
    arr:[2,3,4]
  }
]
```

**Comment by lahmatiy** [comment, marked as answer]
You need to use [`size()`](https://discoveryjs.github.io/jora/#article:jora-syntax-methods-builtin&!anchor=size) method on arrays:
```
.[arr.size() > 1]
```

Try it in [playground](https://discoveryjs.github.io/jora/#report&q=LlthcnIuc2l6ZSgpID4gMV0%3D&graph=eyJjdXJyZW50IjpbMCwwXSwiY2hpbGRyZW4iOlt7ImNoaWxkcmVuIjpbe31dLCJxdWVyeSI6IltcbiAge1xuICAgIGFycjpbMSwyLDNdLFxuICB9LFxuICB7XG4gICAgYXJyOlsxXVxuICB9LFxuICB7XG4gICAgYXJyOlsyLDMsNF1cbiAgfVxuXSJ9XX0%3D)

[Jora documentation](https://discoveryjs.github.io/jora/)
